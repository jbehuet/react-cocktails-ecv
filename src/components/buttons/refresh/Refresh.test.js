import React from "react";
import { create } from "react-test-renderer";

import { Refresh } from "./Refresh";

describe("Refresh button component", () => {
  it("renders correctly", () => {
    const component = create(<Refresh onClick={() => { }} />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
