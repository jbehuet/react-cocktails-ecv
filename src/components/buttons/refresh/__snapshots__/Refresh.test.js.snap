// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Refresh button component renders correctly 1`] = `
<button
  className="refresh"
  onClick={[Function]}
>
  <i
    className="fas fa-redo-alt"
  />
</button>
`;
